routes/transaction.go

```diff
@@ -3968,6 +3997,81 @@ func TransactionSpendingLimitToResponse(
 		}
 	}
 
+	if len(transactionSpendingLimit.StakeLimitMap) > 0 {
```

Comment:
Not sure why this is here. Will take a look and remove...

Reply:
Look up above - this looks like it's duplicated. Confirm that it's the same and then just delete it.

Reply:
I think we can delete all these changes to the `TransactionSpendingLimitToResponse` function.

Reply:
Confirmed they were duplicate and deleted.